tt_metal/impl/device/device.hpp

```diff
@@ -256,6 +256,7 @@ class Device {
     friend bool CloseDevice(Device *device);
 
     // APIs to access this device's work executor
+    bool use_passthrough_scheduling() const;
     void push_work(std::function<void()>&& work, bool blocking = false);
```

Comment:
Methods starting with a verb usually act as a command, not query. 

Can we rename to is_passthrough_scheduling_used or smth like that?

Reply:
Renamed to `can_use_passthrough_scheduling`